src/Http/Plugin/ProblemDetailsPlugin.php

```diff
@@ -18,7 +19,9 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
         /** @var Promise $promise */
         $promise = $next($request);
 
-        return $promise->then(null, function (HttpException $exception) {
+        return $promise->then(null, function (RequestException $exception) {
+            if (!$exception instanceof HttpException) throw $exception;
```

Comment:
Can you expand this line?

`if (!$exception instanceof HttpException) {
    throw $exception;
}`